roles/openshift_health_checker/openshift_checks/__init__.py

```diff
@@ -21,8 +21,16 @@ class OpenShiftCheckException(Exception):
 class OpenShiftCheck(object):
     """A base class for defining checks for an OpenShift cluster environment."""
 
-    def __init__(self, module_executor):
-        self.module_executor = module_executor
+    LOADER_EXCLUDES = (
+        "__init__.py",
+        "mixins.py",
+    )
+
+    def __init__(self, execute_module=None, module_executor=None):
```

Comment:
Yeah, was not the original intention, but I thought it would help @juanvallejo not having to change inflight PRs, and me not having to rebase all the time and update.

For instance, https://github.com/openshift/openshift-ansible/pull/3461 merged yesterday with the old name. Of course the intention is to remove the old name as soon as we have the unit tests for all checks and require new checks to be tested (so that CI would alert us of a bug).

This is a known way on how to do safe refactors :)